.github/workflows/build-ios.beta.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: build-ios-beta
+
+on:
+  push:
+    branches:
+      # TODO: remove the `gha-` to match the current workflow again
+      - build-gha-\d+$
+      - build-gha-ios-\d+$
+      - build-gha-ios-beta-\d+$
+
+env:
+  NODE_VERSION: 16.14.2
+  TERM: xterm
+
+jobs:
+  build-ios-beta:
+    runs-on: macos-12
```

Comment:
Can you share what software is installed with this image?

Reply:
The list is available in [this repo](https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md)